gsa/src/web/pages/extras/feedstatuspage.js

```diff
@@ -265,4 +263,39 @@ const FeedStatus = () => {
   );
 };
 
-export default FeedStatus;
+FeedStatus.propTypes = {
+  feeds: PropTypes.array,
+};
+
+const FeedStatusWrapper = () => {
+  const gmp = useGmp();
+  const [feeds, setFeeds] = useState([]);
+
+  const loadFeeds = () =>
+    // cannot store gmp.feedstatus.readFeedInformation in separate variable
+    // causes undefined "this" in FeedStatus class
```

Comment:
I don't quiet get why we would want to do this in the first place. The comment seems unnecessary.

Reply:
That's because I moved the loadFeeds and setting feeds out of the FeedStatus and into the wrapper. I tried to keep it in there via a variable pointing to gmp.feedstatus.readFeedInformation() but it caused the aforementioned problem. Thought I might explain myself, although if it's clear I can get rid of it.